### DISCUSSION TITLE
How to use localePath with $router.back() or $router.go(-1)?

### DISCUSSION THREAD

**Discussion by mateoKutnjak** [nuxt-modules/i18n#2217] [Q&A]
I can access previous route through `$router.options.history.state.back` but I believe this will mess up history stack. Can I somehow use back() or go(-1) with localePath function?

```
<NuxtLink :to="???"></NuxtLink>
```

**Comment by kazupon** [comment]
This is a question, so we will move to github discussions

**Comment by kazupon** [comment]
I am not sure what exactly you mean by `with` here. If it is asynchronous and `parallel`, I think the order of execution of each depends on the event loop of the javascript.

If seriall order like `localPath` and back() or go(-1) , I think it works.

**Reply by mateoKutnjak** [reply]
I have an element with `@click.prevent="$router.back()"` and I don't know how to use `setLocale` function with `$router.back()` function.

When I change routes with `NuxtLink` I do it whis way to maintain same locale:

```
<NuxtLink
    :to="localePath('/some/route/')"
/>
```

How can I maintain same locale with $router.back() function?
